pkg/logs/sds/scanner_test.go

```diff
@@ -232,6 +232,10 @@ func TestCreateScanner(t *testing.T) {
 		Config: agentConfig,
 	})
 
+	if err != nil {
+		t.Error("no error should happen")
+	}
```

Comment:
Would it make sense to use `require.NoError` here?

Reply:
Converted all new unit tests to use `testify.Require`